jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -211,6 +214,10 @@
 	/*[PR Jazz 85476] Address locking contention on classRepository in getGeneric*() methods */
 	private transient ClassRepositoryHolder classRepoHolder;
 	
+/*[IF JAVA_SPEC_VERSION >= 12]*/
```

Comment:
Why >= 12?  Java 11 needs to have this check as well.

Reply:
As mentioned at https://github.com/eclipse/openj9/issues/11523#issuecomment-754299741, the check is ignored on OpenJDK11/Hotspot when calling `newInstance()` and enabled since OpenJDK12/Hotspot:
```
public class Example {

    public static void main(String[] args) throws Exception {
        new Example().foo();
    }

    @SuppressWarnings("deprecation")
    void foo() throws Exception {
        String provider = Security.getProperty("policy.provider");
        System.out.println("provider = " + provider);
        Object obj = null;
        Class<?> clzz = null;
        
        clzz = Class.forName(provider);
        obj = clzz.newInstance();
        System.out.println("obj = " + obj);
    }
}

$ jdk11_hotspot/bin/java  -version  
openjdk version "11.0.10" 2021-01-19
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.10+8-202101071822)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.10+8-202101071822, mixed mode)


$ ../jdk11_hotspot/bin/java --illegal-access=warn   Example.java
provider = sun.security.provider.PolicyFile
obj = sun.security.provider.PolicyFile@245b4bdc


$ ../jdk11_hotspot/bin/java --illegal-access=debug   Example.java
provider = sun.security.provider.PolicyFile
obj = sun.security.provider.PolicyFile@245b4bdc


$ jdk12_hotspot/bin/java   -version
openjdk version "12.0.2" 2019-07-16
OpenJDK Runtime Environment AdoptOpenJDK (build 12.0.2+10-201911200946)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 12.0.2+10-201911200946, mixed mode, sharing)


$ ../jdk12_hotspot/bin/java --illegal-access=warn   Example.java
provider = sun.security.provider.PolicyFile
WARNING: Illegal reflective access by Example (file:Example.java) to constructor sun.security.provider.PolicyFile()
obj = sun.security.provider.PolicyFile@46238e3f


$ ../jdk12_hotspot/bin/java --illegal-access=debug   Example.java
provider = sun.security.provider.PolicyFile
WARNING: Illegal reflective access by Example (file:Example.java) to constructor sun.security.provider.PolicyFile()
	at Example.foo(Example.java:18)
	at Example.main(Example.java:7)
	at jdk.compiler/com.sun.tools.javac.launcher.Main.execute(Main.java:415)
	at jdk.compiler/com.sun.tools.javac.launcher.Main.run(Main.java:192)
	at jdk.compiler/com.sun.tools.javac.launcher.Main.main(Main.java:132)
obj = sun.security.provider.PolicyFile@32464a14

```
So we should match the RI's behaviour at this point.